x-pack/winlogbeat/module/sysmon/config/winlogbeat-sysmon.js

```diff
@@ -811,6 +811,11 @@ var sysmon = (function () {
                     from: "winlog.event_data.SourceIp",
                     to: "source.ip",
                     type: "ip",
+                },
+				{
```

Comment:
Can you fix the indentation here please; tabs to 4×space.

Reply:
@efd6, Fixed!